tracer/test/Datadog.Trace.IntegrationTests/DiagnosticListeners/AspNetCoreEndpointRoutingTestData.cs

```diff
@@ -37,13 +37,19 @@ public static class AspNetCoreEndpointRoutingTestData
             { "/I/dont/123/exist/", 404, false, "GET /i/dont/?/exist/", EmptyTags() },
             { "/Home/Error", 500, true, "GET Home/Error", EmptyTags() },
             { "/Home/UncaughtError", 500, true, "GET Home/UncaughtError", EmptyTags() },
+#if !NET6_0
```

Comment:
How come these don't work? That seems like a problem? 😕

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/4a467862f211c03a29c558ab1e0154286161ab0e